src/rdo.rs

```diff
@@ -404,6 +410,15 @@ pub fn rdo_mode_decision(
     mode_contexts.push(cw.find_mvrefs(bo, ref_frames, &mut mv_stack, bsize, false, fi, false));
 
     if fi.frame_type == FrameType::INTER {
+      let mut pmv = [MotionVector{ row: 0, col: 0 }; 2];
```

Comment:
So, previously pmv[] was defined for seven different reference frame but now two only. I wonder what kinds impact by this change, if any?

Reply:
Oh, I read it wrong. pmvs[] has still slots for seven ref. While pmv[] has had two, for two references, and no change.

Reply:
Oh, wrong again. The dimension of pmvs was [5][8]